### DISCUSSION TITLE
Integrating  pymupdf4llm with langchain?

### DISCUSSION THREAD

**Discussion by tlevyaleia** [pymupdf/pymupdf4llm#96] [General]
Hello,
Congrats for the great work!
Have you already thought/started discussing about integrating  pymupdf4llm with langchain? 
Something like having a loader class that would produce langchain documents?
Thanks,
Thom

**Comment by JorjMcKie** [comment]
We won't go out hunting for these alternatives.
But we will consider proper PRs offering such solutions.
